src/App.tsx

```diff
@@ -1,24 +1,258 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import {
+  FC,
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import classNames from 'classnames';
+import { Todo } from './types/Todo';
+import { addTodo, getTodos, removeTodo } from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Loader } from './components/Loader';
+import { FilterBy } from './types/FilteredBy';
+import { TodoFilter } from './components/TodoFilter';
+import { Form } from './components/Form';
+import { ErrorMessage } from './components/ErrorMessage';
 
-const USER_ID = 0;
+const USER_ID = 6894;
 
-export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+function filterTodosByCompleted(todos: Todo[], filterBy: string): Todo[] {
```

Comment:
make filterBy type FilterBy

Reply:
I already have this type, i just forgot)